java/build.gradle

```diff
@@ -47,10 +47,10 @@ dependencies {
     implementation("com.amazonaws:aws-lambda-java-events:2.2.1")
 
     // lambda logging
-    implementation("com.amazonaws:aws-lambda-java-log4j2:1.3.0")
-    implementation("org.apache.logging.log4j:log4j-slf4j18-impl:2.15.0")
-    implementation("org.apache.logging.log4j:log4j-core:2.15.0")
-    implementation("org.apache.logging.log4j:log4j-api:2.15.0")
+    implementation("com.amazonaws:aws-lambda-java-log4j2:1.4.0")
+    implementation("org.apache.logging.log4j:log4j-slf4j18-impl:2.16.0")
+    implementation("org.apache.logging.log4j:log4j-core:2.16.0")
+    implementation("org.apache.logging.log4j:log4j-api:2.16.0")
```

Comment:
i would assume that core already depends on api, probably can remove this line if you so wish

Reply:
Good catch, one dep less to care about.